userguide/system.rst

```diff
@@ -891,34 +893,36 @@ shown in
    | Setting              | Value                | Description                                                                                     |
    |                      |                      |                                                                                                 |
    +======================+======================+=================================================================================================+
-   | From E-mail          | string               | the envelope **From** address shown in the email; this can be set to assist with filtering      |
-   |                      |                      | mail on the receiving system                                                                    |
+   | From E-mail          | string               | The envelope **From** address shown in the email. This is set to assist with filtering          |
+   |                      |                      | mail on the receiving system.                                                                   |
+   |                      |                      |                                                                                                 |
    +----------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Outgoing Mail Server | string or IP address | hostname or IP address of SMTP server to use for sending this email                             |
+   | Outgoing Mail Server | string or IP address | Hostname or IP address of SMTP server to use for sending this email.                            |
    |                      |                      |                                                                                                 |
```

Comment:
s/of SMTP server to use for sending this email./of the SMTP server to use for sending email./